### ISSUE TITLE
Pokemon on heatmap not showing up

### ISSUE THREAD

**Issue by daangroot** [RocketMap/RocketMap#2307]
## Expected Behavior

Heatmap should show pokemon.

## Current Behavior

Pokemon on heatmap are not showing up. Using the developer console on Chrome I got two errors.
First when opening the stats page:
```
Uncaught TypeError: Cannot read property 'appendChild' of null
    at app.min.js:5
    at app.min.js:5
```
And the next error occured when opening the heatmap:
```
map.common.min.js:5 Uncaught ReferenceError: isNotifyPoke is not defined
    at setupPokemonMarkerDetails (map.common.min.js:5)
    at setupPokemonMarker (map.common.min.js:5)
    at Object.processAppearance (statistics.min.js:5)
    at Function.each (jquery.min.js:2)
    at Object.<anonymous> (statistics.min.js:5)
    at i (jquery.min.js:2)
    at Object.fireWith [as resolveWith] (jquery.min.js:2)
    at A (jquery.min.js:4)
    at XMLHttpRequest.<anonymous> (jquery.min.js:4)
```
## Possible Solution

-

## Steps to Reproduce (for bugs)

1.  Open the heatmap...

## Your Environment

Client: Windows 10, Google Chrome Version 61.0.3163.100.
Server: Ubuntu 16.04, Python 2.7.

**Comment by sirlindqvist** [comment]
Can confirm I have this issue as well.

**Comment by ChrisSsSsSsS** [comment]
I added a PR that fixes this issue: #2312
